static/sass/components/_intro.scss

```diff
@@ -0,0 +1,64 @@
+// Intro Blocks
+// ============
+// Two column (on wide screens) intro and aside
+// Can be seen on case studies
+
+// wrapping container for intro
+.intro-block {
+  @include grid;
```

Comment:
If something is display grid (after display flex, which this does compile to that order), it will ignore the flex settings, but keep the box-alignment settings.